### PR TITLE
Search stlye edits

### PR DESCRIPTION
**Pull request by CalvinFronda**

### CODE REVIEW

**Comment by NeilAllavarpu** [comment]
@dhuangg check if this overrides/conflicts with anything you currently have

File: src/components/FeatureViews/Search/UpcomingSessionCards.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import Chip from "@material-ui/core/Chip";
+import CardActions from "@material-ui/core/CardActions";
+import { withRouter } from "react-router-dom";
+
+
+
+function UpcomingSessionCards(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/UpcomingSessionCards.js]: Is this for future use? If not, remove the unused state

File: src/reducers/initialState.js [hunk 2]
```diff
@@ -454,7 +454,7 @@ export default {
                 description: "This course is primarily for sophomores and juniors preparing for the SAT.",
                 room_id: 4,
                 type: "C",
-                roster: [23, 99,],
```
Comment (NeilAllavarpu) [inline src/reducers/initialState.js]: Why were rosters changed?

File: src/components/FeatureViews/Search/UpcomingSessionCards.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        <Grid item xs={12} sm={3} className={"UpcomingSessionCards"} style={{ padding: "20px" }}>
+            <Card key={props.user.user_id}
+                style={{ cursor: "pointer" }}
+                onClick={(event) => {
+                    event.preventDefault();
+                    goToRoute(`/${props.user.role}/${props.user.user_id}`);
+                }}>
+                <Grid container>
+                    <Grid item sm={12}>
+                        <Typography align={"center"} variant={"h5"}> Noveber 12, 2019 | 7:00 - 9:00 </Typography>
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/UpcomingSessionCards.js]: Even for testing, retrieve a course from redux and display it by that course's data - easier to plug in actual search results, and easier to test with different data.

File: src/components/FeatureViews/Search/SearchResults.js [hunk 4]
```diff
@@ -1,24 +1,106 @@
...
+import CoursesCards from "./CoursesCards"
 
 
-const SearchResults = ()=>{
-    const [query, setQuery ] = useState('');
+const SearchResults = (props) => {
+    const [query, setQuery] = useState('');
     const [primaryFilter, setPrimaryFilter] = useState("All");
+    const [showItems, setShowItems] = useState(2)
+    const [userList, setUserList] = useState(props.instructors)
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/SearchResults.js]: Is this for future use? If not, remove the unused state

File: src/components/FeatureViews/Search/CoursesCards.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+import Chip from "@material-ui/core/Chip";
+import CardActions from "@material-ui/core/CardActions";
+import { withRouter } from "react-router-dom";
+
+
+
+function CourseCards(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/CoursesCards.js]: Is this for future use? If not, remove the unused state

File: src/components/FeatureViews/Search/CoursesCards.js [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+    const goToRoute = (route) => {
+        props.history.push(props.match.url + route);
+    }
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/CoursesCards.js]: You can make this a hook in a separate file, and then import it in to the places where it is used.

File: src/components/FeatureViews/Search/UpcomingSessionCards.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+
+function UpcomingSessionCards(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
+
+    const goToRoute = (route) => {
+        props.history.push(props.match.url + route);
+    }
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/UpcomingSessionCards.js]: Reused function, possible hook use

File: src/components/FeatureViews/Search/CoursesCards.js [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+        <Grid item xs={12} sm={3} className={"CourseCards"} style={{ "padding": "20px" }}>
+            <Card key={props.user.user_id}
+                style={{ cursor: "pointer", width: "100%" }}
+                onClick={(event) => {
+                    event.preventDefault();
+                    goToRoute(`/${props.user.role}/${props.user.user_id}`);
+                }}>
+                <Grid container style={{ "paddingLeft": "1em" }}>
+                    <Grid item sm={12}>
+                        <Typography align={"left"} variant={"h5"}> Calculus 101 </Typography>
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/CoursesCards.js]: Use a course object from initialState to render data!

File: src/components/FeatureViews/Search/AccountsCards.js [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+        let colour = "#";
+
+        for (i = 0; i < 3; i += 1) {
+            const value = (hash >> (i * 8)) & 0xff;
+            colour += `00${value.toString(16)}`.substr(-2);
+        }
+        /* eslint-enable no-bitwise */
+
+        return colour;
+    }
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/AccountsCards.js]: There should be an `accountUtils.js` file in Accounts that exports this function, import that instead.

File: src/components/FeatureViews/Search/AccountsCards.js [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+
+
+function ProfileCard(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
+
+    const goToRoute = (route) => {
+        props.history.push(props.match.url + route);
+    }
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/AccountsCards.js]: Possible hook use (reused function)

File: src/components/FeatureViews/Search/AccountsCards.js [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+
+import Avatar from "@material-ui/core/Avatar";
+
+
+
+
+function ProfileCard(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/AccountsCards.js]: Is this for future use? If not, remove the unused state

File: src/reducers/initialState.js (hunk 2 above)
Comment (CalvinFronda) [reply src/reducers/initialState.js]: The student tabs won't work because I still need to fix the enrollments object. This is a quick fix

**Pull request closed without merging**